src/previews.ts

```diff
@@ -26,6 +27,7 @@ export const previews: PreviewFlags = {
   frameworkawareness: false,
   functionsparams: false,
   crossservicerules: false,
+  skipnoopfunctiondeploy: false,
```

Comment:
*nit - just for clarity, I'd suggest flag that starts with `functions` as a prefix - e.g. `functionsnoopdeploy`

Reply:
My read on this is `functionsnoopdeploy` suggests we are enabling no-op deploys. What we are trying to do is to enable skipping no-op deploys.

Maybe  `skipdeployingnoopfunctions` ?

Reply:
(Just playing with words here. I don't really care that much about preview flag words. I'd consult with @egilmorez or @jhuleatt if you think we will eventually expose this feature flag to our customers.)

Here's my definition:

**no-op deploy**: Feature to detect and skip deployment of functions that hasn't changed since last deployment

So to me, skipping 'no-op deploy' means flag to disable no-op deploy which is why it's confusing to me unless this is actually what you are trying to do here!